recipes-devtools/python/python3-usbmuxctl_git.bb

```diff
@@ -0,0 +1,28 @@
+DESCRIPTION = "Control software for Linux Automation GmbH USB-Mux that enables automated testing of embedded USB devices."
+HOMEPAGE = "https://github.com/linux-automation/usbmuxctl"
+LICENSE = "LGPL-2.1+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4fbd65380cdd255951079008b364516c"
+
+RDEPENDS_${PN} = " \
+    python3-pyusb \
+    python3-termcolor \
+"
+
+SRC_URI = " \
+    git://github.com/linux-automation/usbmuxctl.git;branch=master \
+    file://99-usbmux.rules \
+    "
+
+# Version 0.1.1
+SRCREV = "096905f9de6df1f396582b8de1faef31d43d1a81"
```

Comment:
Just a minor note, but: Is it intended to use _git.bb suffix for some and _<version>.bb suffix for other recipes when both point to a released version?